src/tools/gophish_import.py

```diff
@@ -53,10 +53,10 @@ def load_landings(api, assessment):
             new_page.redirect_url = page["redirect_url"]
 
         # Debug page information
-        logging.debug("Page Name: {}".format(new_page.name))
-        logging.debug("Redirect ULR: {}".format(new_page.redirect_url))
-        logging.debug("Capture Credentials: {}".format(new_page.capture_credentials))
-        logging.debug("Capture Passwords: {}".format(new_page.capture_passwords))
+        logging.debug("Page Name: %s", new_page.name)
+        logging.debug("Redirect ULR: %s", new_page.redirect_url)
+        logging.debug("Capture Credentials: %s", new_page.capture_credentials)
+        logging.debug("Capture Passwords: %s", new_page.capture_passwords)
```

Comment:
Please take care of the code scanning error here.  If it's a false positive then please document that with a comment.

Reply:
What kind of code scanning error are you encountering here?

We just used this branch for a demo, and it worked as we expected. I havent seen any irregularities from the editor either.

Reply:
Ah okay, I found the issue. 

I dont know why the credentials were being logged like they were. I removed the offending lines for now, and if they become necessary at some point in the future we can revisit adding them in and figuring out some solution to log them in an encrypted manner. 

See a65955c